pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-norecursedirs=third_party/*
\ No newline at end of file
+norecursedirs = third_party/* .venv/* docs/vendor/* docs/_site/*
```

Comment:
This change is unrelated to the main one but speeds up running tests by 3-8 seconds.